src/views/product-integrations-landing/components/paginated-integrations-list/index.tsx

```diff
@@ -0,0 +1,196 @@
+import { IconArrowLeft16 } from '@hashicorp/flight-icons/svg-react/arrow-left-16'
+import { IconArrowRight16 } from '@hashicorp/flight-icons/svg-react/arrow-right-16'
+import classNames from 'classnames'
+import { Integration } from 'lib/integrations-api-client/integration'
+import { useEffect, useState } from 'react'
+import IntegrationsList from '../integrations-list'
+import s from './style.module.css'
+
+interface PaginatedIntegrationsListProps {
+	integrations: Array<Integration>
+	itemsPerPage?: number
+}
+
+export default function PaginatedIntegrationsList({
+	integrations,
+	itemsPerPage = 8,
+}: PaginatedIntegrationsListProps) {
+	// Sort integrations alphabetically
+	const sortedIntegrations = integrations.sort(
+		(a: Integration, b: Integration): number => {
+			if (a.name < b.name) {
+				return -1
+			} else if (a.name > b.name) {
+				return 1
+			} else {
+				return 0
+			}
+		}
+	)
```

Comment:
Does the hardcoded alphabetical sorting prevent us from doing other sorting methods? — i.e. by "created date", by "most favorited", etc.?

Either way, no action needed, just asking for clarity, and to maybe leave a comment for the intent/expected limitations here.

Reply:
It does, but this is the desired sort for now.

In the event in the future we want to add other sorting methods, we would change this here at this level.